dotnet/samples/KernelSyntaxExamples/Example75_AgentTools.cs

```diff
@@ -0,0 +1,168 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.SemanticKernel.Experimental.Agents;
+
+// ReSharper disable once InconsistentNaming
+/// <summary>
+/// Showcase usage of code_interpreter and retrieval tools.
+/// </summary>
+public static class Example75_AgentTools
+{
+    /// <summary>
+    /// Specific model is required that supports agents and parallel function calling.
+    /// Currently this is limited to Open AI hosted services.
+    /// </summary>
+    private const string OpenAIFunctionEnabledModel = "gpt-4-1106-preview";
+
+    // Track agents for clean-up
+    private static readonly List<IAgent> s_agents = new();
+
+    /// <summary>
+    /// Show how to combine coordinate multiple agents.
+    /// </summary>
+    public static async Task RunAsync()
+    {
+        Console.WriteLine("======== Example75_AgentTools ========");
+
+        if (TestConfiguration.OpenAI.ApiKey == null)
+        {
+            Console.WriteLine("OpenAI apiKey not found. Skipping example.");
+            return;
+        }
+
+        // Run agent with code-interpreter tool
```

Comment:
```suggestion
        // Run agent with code interpreter tool
```

Reply:
if we want to get nitty...its `code_interpreter`